app/_includes/components/entity_example/format/snippets/kic.md

```diff
@@ -3,20 +3,40 @@
 {% else %}
 
 ```yaml
+echo "
 {{ include.presenter }}
+" | kubectl apply -f -
 ```
 {% endif %}
 
 {% if include.presenter.entity_type == "plugin" and include.presenter.foreign_keys.size > 0 %}
 Next, apply the `KongPlugin` resource by annotating the {{ include.presenter.targets }}:
 
+
+{% if include.presenter.foreign_keys[0] != "Route" %}
 ```bash
-{% for key in include.presenter.foreign_keys -%}
-kubectl annotate {% if key == "Service" %}service{% elsif key == "Route" %}ingress{% else %}Kong{{ key }}{% endif %} {{ key | upcase}}_NAME konghq.com/plugins={{ include.presenter.data.name }}
+{% for key in include.presenter.foreign_keys -%}m
+{% capture target_name %}{{ key | upcase }}_NAME{% endcapture -%}
+{% if include.presenter.foreign_key_names[key] %}{% assign target_name = include.presenter.foreign_key_names[key] %}{% endif %}
+kubectl annotate {% if key == "Service" %}service{% else %}Kong{{ key }}{% endif %} {{ target_name }} konghq.com/plugins={{ include.presenter.other_plugins }}{{ include.presenter.data.name }}{% if include.presenter.other_plugins %} --overwrite{% endif %}
 {% endfor -%}
 ```
+{% else %}
+{% navtabs api %}
+{% navtab "Gateway API" %}
+```bash
+kubectl annotate -n kong httproute {{ include.presenter.foreign_key_names['route'] }} konghq.com/plugins={{ include.presenter.other_plugins }}
```

Comment:
Is it fine to leave a hanging `=` when there are no other plugins:
![Screenshot 2025-04-07 at 11 41 12 PM](https://github.com/user-attachments/assets/c80b04bc-1c97-4507-a257-0166adf22662)

Reply:
This is a bug. I’ll fix